# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1771060910 -39600
# Parent  ed787827e6628fc1adb68624a3f0831a01487af7
Misc: Remove unneeded space in filter editor panel.

diff --git a/mail/themes/shared/mail/filterEditor.css b/mail/themes/shared/mail/filterEditor.css
--- a/mail/themes/shared/mail/filterEditor.css
+++ b/mail/themes/shared/mail/filterEditor.css
@@ -6,9 +6,9 @@
   min-height: 30px;
 }
 
 #applyFiltersSettings {
-  min-height: 150px;
+  min-height: 100px;
 }
 
 #searchTermListBox {
   flex: 1 1 auto;
